pkg/syncthing/client.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2023 The envd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package syncthing
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"net/http"
+	"path"
+	"time"
+)
+
+func NewApiClient() *http.Client {
```

Comment:
```suggestion
func NewAPIClient() *http.Client {
```

Follow the naming convention in this project. Also need to fix others.

Reply:
Ok, i will pay more attention. Thanks for pointing it out.